crates/autopilot/src/domain/fee/policy.rs

```diff
@@ -55,8 +61,8 @@ impl Surplus {
             boundary::OrderClass::Liquidity => None,
             boundary::OrderClass::Limit => {
                 let policy = domain::fee::Policy::Surplus {
-                    factor: self.factor,
-                    max_volume_factor: self.max_volume_factor,
+                    factor: Factor::try_from(self.factor).unwrap(),
```

Comment:
The domain types itself should also use the `Factor`. That way we don't need unwraps here.